src/simplewallet/simplewallet.cpp

```diff
@@ -245,17 +245,18 @@ namespace
   const char* USAGE_FROZEN("frozen <key_image>");
   const char* USAGE_NET_STATS("net_stats");
   const char* USAGE_WELCOME("welcome");
+  const char* USAGE_CONFIGURE("configure");
   const char* USAGE_VERSION("version");
   const char* USAGE_HELP("help [<command>]");
 
-  std::string input_line(const std::string& prompt, bool yesno = false)
+  std::string input_line(const std::string& prompt, const std::vector<std::string> &values = {})
   {
 #ifdef HAVE_READLINE
     rdln::suspend_readline pause_readline;
 #endif
     std::cout << prompt;
-    if (yesno)
-      std::cout << "  (Y/Yes/N/No)";
+    if (!values.empty())
+      std::cout << "  (" + boost::algorithm::join(values, "/") + ")";
```

Comment:
No need to create multiple temporary strings, write each portion directly to `std::cout` with `<<`.